drivers/timer/mchp_xec_rtos_timer.c

```diff
@@ -271,7 +323,7 @@ static void xec_rtos_timer_isr(const void *arg)
 
 uint32_t sys_clock_elapsed(void)
 {
-	return 0U;
+	return 0u;
```

Comment:
please make unrelated style changes in a separate PR

Reply:
Put it back to 0U until future style PR.